src/recipe.cpp

```diff
@@ -94,7 +95,7 @@ int recipe::batch_time( const Character &guy, int batch, float multiplier, size_
 
     // if recipe does not benefit from batching and we have no assistants, don't do unnecessary additional calculations
     if( batch_rscale == 0.0 && assistants == 0 ) {
-        return static_cast<int>( local_time ) * batch;
+        return static_cast<int64_t>( local_time ) * batch;
```

Comment:
I am not sure, just came up with the thought in my mind, will there be potential precision loss here, given that now it's doing 64-bit arithmetic?

Reply:
Do we care that much about the precision? I'm not too concerned about it.

Reply:
Also escalating `local_time` and `total_time` to 64-bit `double` makes more sense to me.

Reply:
Ok. It is done.